models/core-java/src/main/java/org/commonjava/indy/pkg/npm/model/NPMPackageTypeDescriptor.java

```diff
@@ -29,6 +29,8 @@
 {
     public static final String NPM_PKG_KEY = PKG_TYPE_NPM;
 
+    public static final String NPM_METADATA_NAME = "package.json";
```

Comment:
IIRC we have several places to define this constant, like FoloNPMContentAccessResource, PackageMetadataMerger. I think we should consolidate them together into this one.
And some magic string usage of this constant in these classes:
DecoratorUtils#L110
NPMPackageMaskingTransferDecorator#L74
PackageMetadataGenerator##L375

Reply:
Yeah, done to ref the constant instead.